specification/search/data-plane/Microsoft.Azure.Search.Data/preview/2019-05-06-preview/searchindex.json

```diff
@@ -1039,6 +1111,30 @@
       },
       "description": "Specifies the syntax of the search query. The default is 'simple'. Use 'full' if your query uses the Lucene query syntax."
     },
+    "FeaturesMode": {
+      "type": "string",
+      "enum": [
+        "disabled",
+        "enabled"
+      ],
+      "x-ms-enum": {
+        "name": "FeaturesMode",
+        "modelAsString": false
+      },
+      "description": "A value that specifies whether the results should include scoring features, such as per field similarity. The default is 'disabled'. Use 'enabled' to expose additional scoring features."
+    },
+    "ScoringStatistics": {
+      "type": "string",
+      "enum": [
+        "local",
+        "global"
+      ],
+      "x-ms-enum": {
+        "name": "ScoringStatistics",
+        "modelAsString": false
+      },
+      "description": "A value that specifies whether we want to calculate scoring statistics (such as document frequency) globally for more consistent scoring, or locally, for lower latency. The default is 'local'. Use 'global' to aggregate scoring statistics globally before scoring."
```

Comment:
Maybe mention possible perf consequences in the description? Same for sessionID.